src/pages/playground.js

```diff
@@ -1,8 +1,11 @@
 import React, { useState } from 'react';
 import { FaqSection, FlexSection, HeroSection } from '../components';
+import Footer from '../components/marginals/Footer/Footer';
```

Comment:
Line 2 @ratnakar5938 is importing everything from the component module, you're importing relative to the absolute path. Please fix it to one format.